specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Hrm, I have been usually going with the singular on type names even for types that are defined as sets of Ts.  But I'm not particularly bothered by this if it helps with clarity and comprehension.  I'll make the change.